api/README.md

```
@@ -301,16 +301,134 @@ docker build -t bocchi-api:latest .
 docker buildx build --platform linux/amd64,linux/arm64 .
 ```
 
+### Docker Development
+
+#### Local Docker Build
+```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)